tests/imodels-clients-tests/src/integration/management/ChangesetGroupOperations.test.ts

```diff
@@ -0,0 +1,87 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { AuthorizationCallback, GetChangesetGroupListParams, GetSingleChangesetGroupParams, IModelsClient, IModelsClientOptions, IModelsErrorCode } from "@itwin/imodels-client-management";
+import { ReusableIModelMetadata, ReusableTestIModelProvider, TestAuthorizationProvider, TestUtilTypes, assertChangesetGroup, assertCollection, assertError } from "@itwin/imodels-client-test-utils";
+
+import { getTestDIContainer } from "../common";
+
+describe("[Management] ChangesetGroupOperations", () => {
+  let iModelsClient: IModelsClient;
+  let authorization: AuthorizationCallback;
+  let testIModel: ReusableIModelMetadata;
+
+  before(async () => {
+    const container = getTestDIContainer();
+
+    const iModelsClientOptions = container.get<IModelsClientOptions>(TestUtilTypes.IModelsClientOptions);
+    iModelsClient = new IModelsClient(iModelsClientOptions);
+
+    const authorizationProvider = container.get(TestAuthorizationProvider);
+    authorization = authorizationProvider.getAdmin1Authorization();
+
+    const reusableTestIModelProvider = container.get(ReusableTestIModelProvider);
+    testIModel = await reusableTestIModelProvider.getOrCreate();
+  });
+
+  it("should return all items when querying collection", async () => {
+    // Arrange
+    const getChangesetGroupListParams: GetChangesetGroupListParams = {
+      authorization,
+      iModelId: testIModel.id
+    };
+
+    // Act
+    const changesetGroups = iModelsClient.changesetGroups.getList(getChangesetGroupListParams);
+
+    // Assert
+    await assertCollection({
+      asyncIterable: changesetGroups,
+      isEntityCountCorrect: (count) => count === testIModel.changesetGroups.length
```

Comment:
`testIModel.changesetGroups.length` could be zero and this test would succeed. is it asserted somewhere else that there's some changeset groups present? if so, `count > 0` could be sufficient here

Reply:
It would throw an error when retrieving reusable iModel data. Added `count > 0` for a good measure